_adr/92/index.adoc

```diff
@@ -0,0 +1,102 @@
+---
+num: 91
+title: "Toward a more unified UX: BFF"
+status: "Draft"
+authors:
+  - "Andrea Peruffo"
+  - "Riccardo Forina"
+  - "Eric Wittmann"
+tags:
+  - UI
+  - UX
+  - Dev Exp
+applies_padrs: # What PADRs does this ADR apply?
+applies_patterns: # What APs does this ADR apply?
+---
+
+## Context and Problem Statement
+
+The integration of the Enterprise/Hybrid option of RHOSAK in UI, SDKs and CLI has shown that there is a margin for improvement in how we expose and consume our services APIs.
+
+* the current services are offering the bare functionality leaving room for integration code to live in the services' consumer codebases
```

Comment:
I'm not sure I understand this 100%.  Can you explain a bit more what you mean and maybe we can reword a bit to be more clear?

Reply:
Tentatively:
```
the services are exposing APIs for the offered specific functionality, but we lack an integration layer
```

what we are trying to say is that there is no integration logic exposed by the current services.
The services are offering APIs modeled after their specific functionality and are very vertical, but any operation that involves more than one service becomes difficult and involves writing the integration logic on the client side.

Feel free to propose a better working 🙏 !